inst/schema/ModelRunOptions.schema.json

```diff
@@ -2,25 +2,48 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "definitions": {
     "selectControl": {
+      "type": "object",
       "properties": {
         "name": { "type": "string" },
         "label": { "type": "string" },
         "type": { "enum": [ "select", "multiselect" ] },
         "required": { "type": "boolean" },
-        "default": { "type": "string" },
+        "value": { 
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
```

Comment:
This type shouldn't have changed, should just be a string (i.e. the `id` of the option you want selected).